sub-projects/hub/src/server/aws/aws_s3.js

```diff
@@ -0,0 +1,42 @@
+const AWS = require('aws-sdk');
+const async = require('async');
+const bucketName = 'knownfaces';
+const path = require('path');
+const fs = require('fs');
+let pathParams, image, imageName;
+
+const s3 = new AWS.S3({region: 'us-west-1'});
+const createItemObject = (callback) => {
+  const params = {
+    Bucket: bucketName,
+    Key: '${imageName}',
+    ACL: 'public-read',
+    Body: image
+  };
+    s3.putObject(params, function(err, data){
+      if(err){
+        console.log("Error uploading image: ", err);
+        callback(err,NULL);
+      } else {
+        console.log("Successfully uploaded image on S3", data);
+        callback(null, data);
+      }
+    })
+}
+exports.upload = (req, res, next) => {
+  var tmp_path = req.files.file.pathParams;
```

Comment:
Do you have ESLint working on your computer? It typically prefers `let` or `const` to `var`